### PR TITLE
Support VS2015, VC14, and MsBuild14

### PR DESCRIPTION
**Pull request by shoelzer**
This PR is on the "eight" branch. I made a separate PR for "master".

### CODE REVIEW

**Comment by shoelzer** [comment]
I can't figure out why those CI checks failed. Should I expect them to work on the "eight" branch?

What is the easiest way to also get these changes into master? Will "eight" be merged at some point or should I make a separate pull request?

File: master/buildbot/steps/vstudio.py [hunk 1]
```diff
@@ -386,6 +383,13 @@ class VC12(VC11):
 VS2013 = VC12
 
 
+# VC14 doesn't look like it needs extra stuff.
+class VC14(VC12):
+    default_installdir = 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0'
```
Comment (sa2ajj) [inline master/buildbot/steps/vstudio.py]: I understand that this follows the way in other classes, however I'd suggest to use `r'...\...'` to improve readability.

**Comment by sa2ajj** [comment]
Otherwise, LGTM.

**Comment by sa2ajj** [comment]
@shoelzer I'm sorry it took so long to take a look at the change.  Do you think you will be able to address my comment in the coming days?

**Comment by sa2ajj** [comment]
@shoelzer: ping?

**Comment by sa2ajj** [comment]
closed per stalled PR policy

**Pull request closed without merging**